lms/djangoapps/certificates/tests/test_cert_management.py

```diff
@@ -35,11 +36,12 @@ def _create_cert(self, course_key, user, status):
         # Enroll the user in the course
         CourseEnrollmentFactory.create(
             user=user,
-            course_id=course_key
+            course_id=course_key,
+            mode=CourseMode.HONOR
```

Comment:
I've added a test for verified. It doesn't really make sense to test audit, though, since these are tests that have to do with eligible certificates, and all new audit certs are ineligible.